api/kyverno/v1/common_types.go

```diff
@@ -116,12 +115,20 @@ type ConfigMapReference struct {
 }
 
 type APICall struct {
-	// URLPath is the URL path to be used in the HTTP GET request to the
+	// URLPath is the URL path to be used in the HTTP GET or POST request to the
```

Comment:
Not that it's the aim of this PR, but we should probably add a note that filter parameters are also supported in this field. (See last paragraph in the docs [here](https://kyverno.io/docs/writing-policies/external-data-sources/#url-paths).)

Reply:
I added a link to the docs